### PR TITLE
Adds email, lat and lng to park model

### PR DESCRIPTION
**Pull request by TravisGM92**
### Participating Group Members:
* Travis McKinstry 
 
### Code Highlights:
   * Adds attributes to the Park model in `Park`, as well as the tests needed
 
### Have Tests Been Added?
- [ ] No.
- [ ] Yes, but all tests are not passing. 
- [x] Yes, and all are passing.
 
### Any background context you want to provide?
FE wanted extra attributes for Park model upon creation

 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: main_app/migrations/0006_auto_20210113_1531.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            model_name='park',
+            name='lat',
+            field=models.FloatField(default=0.0, max_length=20),
+        ),
+        migrations.AddField(
+            model_name='park',
+            name='lng',
+            field=models.FloatField(default=0.0, max_length=20),
+        ),
+    ]
```
Comment (judithpillado) [inline main_app/migrations/0006_auto_20210113_1531.py:28]: could you tell me the command you used to create the migrations? (Is there a command to add columns like there is for Rails: `rails g migration AddArbitraryToTables arbitrary:references`?)

File: main_app/tests/test_app.py [hunk 2]
```diff
@@ -21,41 +21,41 @@ def test_can_reformat_json_object(self):
...
+    #     "name": "Fake Park",
+    #     "formatted_address": "1234 Fake Street",
+    #     "opening_hours": "never",
+    #     "photo": "too good for photos",
+    #     "rating": "100 mofo"
+    #     }
+    #     data = requests.post('http://localhost:8000/api/park/create/', params)
+    #     self.assertEqual(type(data), requests.models.Response)
+    #
+    #     result = requests.get('http://localhost:8000/api/park/all')
```
Comment (judithpillado) [inline main_app/tests/test_app.py:61]: are they commented out because the tests are not passing?

**Review by judithpillado — APPROVED** [review]

File: main_app/migrations/0006_auto_20210113_1531.py (hunk 1 above)
Comment (TravisGM92) [reply main_app/migrations/0006_auto_20210113_1531.py:28]: Yeah and I added it to the `README.md`, but the commands are (after you make changes to the model):

`python3 manage.py makemigrations`
`python3 manage.py migrate`

File: main_app/tests/test_app.py (hunk 2 above)
Comment (TravisGM92) [reply main_app/tests/test_app.py:61]: Oh crap. They do pass if the server is running, I didn't want to keep running those tests cause they passed. My bad. I'll uncomment them now. Good catch

**Pull request merged**